src/components/FindNamedEntitiesForm.vue

```diff
@@ -78,20 +86,36 @@ export default {
   mixins: [utils],
   props: {
     /**
-     * Callback function to call when the form have been submitted (this should be replaced by an event in future versions).
+     * Dark mode background option
+     */
+    dark: {
+      type: Boolean
+    },
+    /**
+     * Project name to select in the input instead of default project
      */
-    finally: {
-      type: Function,
-      default: noop
+    projectName: {
+      type: String,
+      default: null
+    },
+    /**
+     * Disable project selection select input
+     */
+    disableProjectSelection: {
+      type: Boolean,
+      default: false
     }
   },
   data() {
     return {
       pipelines: [],
-      disabled: false
+      waiting: false
```

Comment:
As far as I can see, `this.waiting` is always false. Maybe you meant to use the `this.$wait.start('load ner pipelines')`?

Reply:
Yes  good catch thanks